apiary.apib

```diff
@@ -2959,11 +2933,14 @@ Shows all metadata associated with the specified column.
                 }
             ]
 
+## Deleting Metadata [/v2/storage/{objectType}/{objectId}/metadata/{metadata_id}]
```

Comment:
i'd put it in each section, the {objectType} parameter is not in the other API calls either